ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -152,6 +152,8 @@
 {% include 'eos/ipv6-routing.j2' %}
 {# Monitor Sessions #}
 {% include 'eos/monitor-sessions.j2' %}
+{# Object Tracking #}
+{% include 'eos/object-tracking.j2' %}
```

Comment:
Need to match the eos cli output.
<img width="355" alt="image" src="https://user-images.githubusercontent.com/88734745/160411987-f3ced4c1-cfbc-4e99-a9f0-267c50a87e29.png">

Please move these two lines below line64.

Reply:
Thanks Tony. So the sorting here should be in accordance to when to command appears in the CLI?

Reply:
Yes.